stan/math/opencl/kernels/matrix_multiply.hpp

```diff
@@ -79,12 +79,33 @@ static const char* matrix_multiply_kernel_code = STRINGIFY(
           // For the tiles on the diagonal we can ignore the values over
           // the diagonal if the matrix is lower triangular or under
           // the diagonal if the matrix is upper triangular
-          A_local[thread_block_col + w * THREAD_BLOCK_SIZE_COL]
-                 [thread_block_row]
-              = A[(tiled_j + w * THREAD_BLOCK_SIZE_COL) * M + i];
-          B_local[thread_block_col + w * THREAD_BLOCK_SIZE_COL]
-                 [thread_block_row]
-              = B[(j + w * THREAD_BLOCK_SIZE_COL) * K + tiled_i];
+          const A_temp_j = tiled_j + w * THREAD_BLOCK_SIZE_COL;
```

Comment:
I'm not sure what the temp things are here, or why they're temp - are there any other words you could use to describe them or put in a comment nearby? is it basically just the current column index? Would a name like `B_curr_col` or `B_curr_j` work? I'm not really sure what is typical in code like this. I was originally going to recommend splitting this out into a function but abstraction is so annoying in OpenCL, the function would have like 10 arguments.

Reply:
Yep, B_curr_col would work and definitely be better. Will rename these.